bciers/apps/reporting/src/app/components/operations/cells/ActionCell.tsx

```diff
@@ -5,10 +5,17 @@ import { createReport } from "@reporting/src/app/utils/createReport";
 import { createReportVersion } from "@reporting/src/app/utils/createReportVersion";
 import { getReportingYear } from "@reporting/src/app/utils/getReportingYear";
 import Button from "@mui/material/Button";
-import { BC_GOV_LINKS_COLOR } from "@bciers/styles";
+import {
+  BC_GOV_LINKS_COLOR,
+  BC_GOV_PRIMARY_BRAND_COLOR_BLUE,
+} from "@bciers/styles";
 import { ReportOperationStatus } from "@bciers/utils/src/enums";
 
-const ActionCell = (params: GridRenderCellParams) => {
+interface ActionCellProps extends GridRenderCellParams {
+  isReportingOpen: boolean;
+}
+
+const ActionCell: React.FC<ActionCellProps> = (params) => {
```

Comment:
I suggest destructuring the params object to appease sonarcloud and be consistent with how react FCs are implemented in the rest of the app

```suggestion
const ActionCell: React.FC<ActionCellProps> = ({row, isReportingOpen}) => {
```